src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.macios.cs

```diff
@@ -37,46 +34,9 @@ public MauiMediaElement(AVPlayerViewController playerViewController, UIViewContr
 			playerViewController.AdditionalSafeAreaInsets =
 				new UIEdgeInsets(insets.Top * -1, insets.Left, insets.Bottom * -1, insets.Right);
 
-			// Add the View from the AVPlayerViewController to the parent ViewController
-			if (viewController is not ShellFlyoutRenderer && viewController is not PageViewController && viewController is not UICollectionViewController)
-			{
-				viewController.AddChildViewController(playerViewController);
-			}
-			viewController.View.AddSubview(playerViewController.View);
-		}
-
-#endif
-
-		AddSubview(playerViewController.View);
-	}
-
-	/// <summary>
-	/// Initializes a new instance of the <see cref="MauiMediaElement"/> class.
-	/// </summary>
-	/// <param name="playerViewController">The <see cref="AVPlayerViewController"/> that acts as the platform media player.</param>
-	/// <exception cref="NullReferenceException">Thrown when <paramref name="playerViewController"/><c>.View</c> is <see langword="null"/>.</exception>
-	public MauiMediaElement(AVPlayerViewController playerViewController)
-	{
-		ArgumentNullException.ThrowIfNull(playerViewController.View);
-		playerViewController.View.Frame = Bounds;
-
-#if IOS16_0_OR_GREATER || MACCATALYST16_1_OR_GREATER
-		// On iOS 16+ and macOS 13+ the AVPlayerViewController has to be added to a parent ViewController, otherwise the transport controls won't be displayed.
-		var viewController = WindowStateManager.Default.GetCurrentUIViewController();
-
-		// If we don't find the viewController, assume it's not Shell and still continue, the transport controls will still be displayed
-		if (viewController?.View is not null)
-		{
-			// Zero out the safe area insets of the AVPlayerViewController
-			UIEdgeInsets insets = viewController.View.SafeAreaInsets;
-			playerViewController.AdditionalSafeAreaInsets =
-				new UIEdgeInsets(insets.Top * -1, insets.Left, insets.Bottom * -1, insets.Right);
 
 			// Add the View from the AVPlayerViewController to the parent ViewController
-			if (viewController is not ShellFlyoutRenderer && viewController is not PageViewController && viewController is not UICollectionViewController)
-			{
-				viewController.AddChildViewController(playerViewController);
-			}
+			viewController.AddChildViewController(playerViewController);
 			viewController.View.AddSubview(playerViewController.View);
```

Comment:
Why we adding Why we adding `playerViewController.View` to parent's view controller view?
In my original code I've removed this as it causes that player area was empty.
I don't think that adding same view twice to view hierarchy is a good idea...

Reply:
Removed as suggested. Tested it and the behavior is as expected. The player controls function with this code removed. TY that was a good catch. Previously removing that line caused player controls to not show.